notification/webhook/sender.go

```diff
@@ -0,0 +1,117 @@
+package webhook
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"net/http"
+	"strconv"
+
+	"github.com/target/goalert/notification"
+)
+
+type Sender struct{}
+
+// POSTDataType is the type of alert data being sent
+type POSTDataType string
+
+const (
+	TypeTest              POSTDataType = "Test"
+	TypeVerification      POSTDataType = "Verification"
+	TypeAlert             POSTDataType = "Alert"
+	TypeAlertBundle       POSTDataType = "AlertBundle"
+	TypeAlertStatus       POSTDataType = "AlertStatus"
+	TypeAlertStatusBundle POSTDataType = "AlertStatusBundle"
+)
+
+// POSTData is a union of all possible message types, should be populated accordingly
```

Comment:
Describe what its purpose/use is, rather than describing the structure. I wonder if calling it `Message` would be the more descriptive name?

For example, `Message is sent as JSON to all webhook endpoints by a Sender. See the Sender.Send method for details on what fields are sent per MessageType.`